.ci/semaphore.sh

```diff
@@ -24,6 +24,66 @@ all-sevntu-checks)
   removeFolderWithProtectedFiles $working_dir
   ;;
 
+check-missing-pitests)
```

Comment:
please move this code to new file `validation.sh`,  lets keep all CI validations in CI agnostic files (as we can execute them in any CI and even on local manually).
We are historically placed commands in CI specific files like "travis.sh, wercker.sh, ...." it was design mistake. Only after many years of managing of this CI set I came to conclusion that it should be like:
```
travis.yml
  -> some-cli-command argument1
  -> some-shell-file.sh command
  -> some-another-big-validation-task.sh 
some-ci.yml
  -> some-cli-command argument2
  -> some-shell-file.sh command2
  -> some-another-big-validation-task2.sh 
```

probably last good example is `pitest.sh` (attention to name, no CI reference). We grouped all same validations in one file, but it can be executed from any CI. I did not have time to refactor all others to same model.

Even some CI validations are specific to CI, it is probably a misdesign of validations. All commands should be executed on local, user should be able to reproduce CI failure on local.

Reply:
done